sdk/synapse/azure-synapse/azure/synapse/_synapse_client.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any
+
+from azure.core import PipelineClient
+from msrest import Deserializer, Serializer
+
+from ._configuration import SynapseClientConfiguration
+from .operations import MonitoringOperations
+from .operations import SparkBatchOperations
+from .operations import SparkSessionOperations
+from .operations import WorkspaceAclOperations
+from . import models
+
+
+class SynapseClient(object):
+    """
+
+    :ivar monitoring: MonitoringOperations operations
+    :vartype monitoring: azure.synapse.operations.MonitoringOperations
+    :ivar spark_batch: SparkBatchOperations operations
+    :vartype spark_batch: azure.synapse.operations.SparkBatchOperations
+    :ivar spark_session: SparkSessionOperations operations
+    :vartype spark_session: azure.synapse.operations.SparkSessionOperations
+    :ivar workspace_acl: WorkspaceAclOperations operations
+    :vartype workspace_acl: azure.synapse.operations.WorkspaceAclOperations
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: azure.core.credentials.TokenCredential
+    :param synapse_dns_suffix: Gets the DNS suffix used as the base for all Synapse service requests.
+    :type synapse_dns_suffix: str
+    :param livy_api_version: Valid api-version for the request.
+    :type livy_api_version: str
+    """
+
+    def __init__(
+        self,
+        credential,  # type: "TokenCredential"
+        synapse_dns_suffix,  # type: str
```

Comment:
Can we set default value "dev.azuresynapse.net" for synapse_dns_suffix? Just like livy_api_version

Reply:
fixed